spec/core_ext/object/descendants_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Class do
+  before do
```

Comment:
Can you use anonymous classes for this?  Also, we should probably test `Module` too.

```ruby
let(:first) { Class.new }
let(:second) { Class.new(first) }
```